docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+version: '3.8'
+
+services:
+  mysqldb:
+    platform: linux/x86_64
+    image: mysql
+    restart: unless-stopped
+    env_file: ./.env
+    environment:
+      - MYSQL_ROOT_PASSWORD=${MYSQLDB_ROOT_PASSWORD}
+      - MYSQL_DATABASE=${MYSQLDB_DATABASE}
+    ports:
+      - ${MYSQLDB_LOCAL_PORT}:${MYSQLDB_DOCKER_PORT}
+    healthcheck:
+      test: [ "CMD", "mysqladmin", "ping", "-h", "localhost" ]
+      interval: 30s
+      timeout: 30s
+      retries: 3
+  app:
+    platform: linux/x86_64
+    depends_on:
+      - mysqldb
+    restart: on-failure
+    image: online-book-store
+    build: .
+    env_file: ./.env
+    ports:
+      - ${SPRING_LOCAL_PORT}:${SPRING_DOCKER_PORT}
+      - ${DEBUG_PORT}:${DEBUG_PORT}
+    environment:
+      SPRING_APPLICATION_JSON: '{
+        "spring.datasource.url": "jdbc:mysql://mysqldb:${MYSQLDB_DOCKER_PORT}/${MYSQLDB_DATABASE}?serverTimeZone=UTC",
+        "spring.datasource.username": "${MYSQLDB_USER}",
+        "spring.datasource.password": "${MYSQLDB_ROOT_PASSWORD}",
+        "spring.datasource.driver-class-name" : "com.mysql.cj.jdbc.Driver",
+
+         "spring.jpa.hibernate.ddl-auto": "validate",
+         "spring.jpa.show-sql": "true",
+         "spring.jpa.open-in-view": "false",
+         "server.servlet.context-path": "/api",
+         "spring.liquibase.enabled": "true",
+
+         "jwt.expiration": "300000000",
+         "jwt.secret": "hellomates23r43r8938fywefwefwefwsegty2hofhlwfiulai"

```

Comment:
please consider a more secure approach :) When you share secrets on github everyone can steal them. In this case even .env files are more preferable